### PR TITLE
修正工作流构件无法下载的问题

### PR DESCRIPTION
**Pull request by Silverteal**
Github Actions 的构件系统对非ASCII字符的支持有些神经，会导致构件无法下载。
此PR暂时缓解了这个问题……

### CODE REVIEW

**Comment by allMagicNB** [comment]
6

**Comment by Deep-Dark-Forest** [comment]
要不要加一行让如果上传文件失败就变成 error？

**Comment by Silverteal** [comment]
> 要不要加一行让如果上传文件失败就变成 error？

1.上传失败本来就会出错
2.上传是正常的，有问题的是下载流程

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -69,11 +71,12 @@ jobs:
...
 
-    # 第四步，上传编译产物到Artifacts
-    - name: 上传编译产物
+    # 第四步，打包并上传编译产物到Artifacts
+    - name: 打包并上传编译产物
       uses: actions/upload-artifact@v4
       with:
-        # Artifact压缩包名称，此处命名为友好名称_原始名称
-        name: ${{ matrix.name }}_${{ matrix.configuration }}
+        # Artifact压缩包名称，此处命名为友好名称
```
[the remark this answers is not in the document]
Comment (Silverteal) [reply .github/workflows/build.yml:78]: > Is the second half of the sentence still necessary when only English is used as the name?

In fact I didn't want to add the second part at the first time, but someone suggested doing so. I'm not going to use the second part, at least here when Chinese name is not included.

**Comment by Silverteal** [comment]
> Personally, I really do not see any projects using non-ASCII characters in GHA artifact file names. And now it is indeed impossible to download. I remember trying to download it before, but I do not remember if it worked properly. So this PR is very necessary.
> 
> Speaking of this, I just happen to have some opinions on the GHA of this project, so I will digress.
> 
> Could we consider adding information about the commit hash that triggered the build in the artifact package file name? This allows you to download multiple artifacts of different builds at the same time without overwriting them, and to distinguish which commit’s build an artifact contains. The best practice I have seen within the Minecraft community is Prism Launcher.
> 
> I personally agree with your idea of ​​uploading GHA artifacts to Releases. However, there are still many forms. Some projects upload artifacts to a fixed tag, and each time a new build is triggered, the previous artifacts are overwritten; some projects create a new tag every time a build is triggered and upload the artifacts to it. I would like to hear your opinion.

In my mind, using release instead of artifact is only for the reason that artifacts can't use non-ascii names. So more configurations is not under my consideration. Due to the fact that Github is not the major method for PCL's distributing, maintaining a system of releases seems to pay more than its benefits.

**Review by LTCatt — COMMENTED** [review]
因为我确实不熟 GHA，所以 review 就交给大家了 orz

**Comment by LTCatt** [comment]
fixed by #4009

**Pull request closed without merging**